e2e/cypress/integration/plugins/marketplace/marketplace_spec.js

```diff
@@ -300,4 +330,47 @@ describe('Plugin Marketplace', () => {
             cy.get('#marketplace-plugin-github').should('be.visible');
         });
     });
+
+    describe('EnableRemoteMarketplace disabled, should', () => {
+        beforeEach(() => {
+            const newSettings = {
+                PluginSettings: {
+                    Enable: true,
+                    EnableMarketplace: true,
+                    EnableRemoteMarketplace: false,
+                    MarketplaceUrl: 'https://api.integrations.mattermost.com',
+                },
+            };
+            cy.apiUpdateConfig(newSettings);
+
+            // # Login as sysadmin
+            cy.apiLogin('sysadmin');
+            cy.visit('/');
+
+            // # Click hamburger main menu
+            cy.get('#sidebarHeaderDropdownButton').click();
+
+            // # Open up marketplace modal
+            cy.get('#marketplaceModal').click();
+        });
+
+        it('not display any plugins and no error bar', () => {
+            // * no plugins should be visible
+            cy.get('#marketplaceTabs-pane-allPlugins').find('.more-modal__row').should('have.length', 0);
+
+            // * no error bar should be visible
+            cy.get('#error_bar').should('not.be.visible');
```

Comment:
We're doing this in several places (and will do clean up soon) but if the element is not rendered in DOM, better to check its existence.
```suggestion
            cy.get('#error_bar').should('not.exist');
```
With `not.be.visible`,
![Screen Shot 2019-12-14 at 12 47 51 AM](https://user-images.githubusercontent.com/5334504/70819794-0c21ae80-1e12-11ea-80d2-cc6734f64788.png)
With `not.exist`,
![Screen Shot 2019-12-14 at 12 47 34 AM](https://user-images.githubusercontent.com/5334504/70819818-19d73400-1e12-11ea-9c46-2c5affc11d9c.png)